tools/EventClients/Clients/PS3 BD Remote/ps3_remote.py

```diff
@@ -95,7 +95,7 @@ def get_remote_address(remote, target_name = "BD Remote Control"):
                 remote.connect((target_address,19))
                 target_connected = True
                 print "Remote Paired.\a"
-                xbmc.send_notification("Pairing Successfull",
```

Comment:
I'm not saying you don't/shouldn't. Just pleading not to be forced to do it in this PR (or any of my current PRs). -- You rightfully objected to taking my initial series because it was too big.

Commits/PRs should be focussed on individual tasks.